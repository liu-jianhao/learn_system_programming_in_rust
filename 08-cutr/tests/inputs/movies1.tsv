title	year	director
The Blues Brothers	1980	John Landis
Les Mis√©rables	2019	Tom Hooper
