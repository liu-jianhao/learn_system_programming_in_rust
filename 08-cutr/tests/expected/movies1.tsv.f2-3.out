year	director
1980	John Landis
2019	Tom Hooper
